### ISSUE TITLE
Sequence Diagram #2: Registration

### ISSUE THREAD

**Issue by oguzhekim** [bounswe/bounswe2024group7#33]
We should create a sequence diagram of an user trying to login, which is one of the use cases.

**Comment by MertCengiz** [comment]
The first version of the First Sequence Diagram related to registration is avaliable [here](https://github.com/bounswe/bounswe2024group7/blob/main/Sequence%20Diagram-2-Registration-Of-A-New_User.mmd). Review needed and highly recommended.

**Comment by oguzhekim** [comment]
I have revised the registration sequence diagram according to the feedback from TA. These are the changes I have made:
* Removed GuestUser and added Interface and System instead and adjusted alternative conditions accordingly.
* Showed the communication between Interface-System and System-Database.
* Wrote the communication between the actor and interface in an abstract manner.
* Added explanatory notes for responses returned.

The new diagram is available [here](https://github.com/bounswe/bounswe2024group7/wiki/Sequence-Diagrams), along with the other sequence diagrams.
